e2e/playground.py

```diff
@@ -0,0 +1,2135 @@
+        #     self.date_start.expect_value(start_val, timeout=timeout)
+        #     self.date_end.expect_value(end_val, timeout=timeout)
+
+    # min: Optional[Union[date, str]] = None,
+    def expect_min_date(
+        self,
+        value: typing.Union[datetime.date, AttrValue],
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_min_date(value, timeout=timeout)
+        self.date_end.expect_min_date(value, timeout=timeout)
+
+    # max: Optional[Union[date, str]] = None,
+    def expect_max_date(
+        self,
+        value: typing.Union[datetime.date, AttrValue],
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_max_date(value, timeout=timeout)
+        self.date_end.expect_max_date(value, timeout=timeout)
+
+    # format: str = "yyyy-mm-dd",
+    def expect_format(
+        self,
+        value: AttrValue,
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_format(value, timeout=timeout)
+        self.date_end.expect_format(value, timeout=timeout)
+
+    # startview: str = "month",
+    def expect_startview(
+        self,
+        value: AttrValue,
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_startview(value, timeout=timeout)
+        self.date_end.expect_startview(value, timeout=timeout)
+
+    # weekstart: int = 0,
+    def expect_weekstart(
+        self,
+        value: typing.Union[int, AttrValue],
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_weekstart(value, timeout=timeout)
+        self.date_end.expect_weekstart(value, timeout=timeout)
+
+    # language: str = "en",
+    def expect_language(
+        self,
+        value: AttrValue,
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        # These values should be the same, so checking both independently seems fair
+        self.date_start.expect_language(value, timeout=timeout)
+        self.date_end.expect_language(value, timeout=timeout)
+
+    # separator: str = " to ",
+    def expect_separator(
+        self,
+        value: TextValue,
+        *,
+        timeout: Timeout = None,
+    ) -> None:
+        if value is None:
+            value = ""
+        playwright_expect(self.loc_separator).to_have_text(value, timeout=timeout)
+
+    # width: Optional[str] = None,
+
+    # autoclose: bool = True,
+    def expect_autoclose(
+        self,
+        value: bool,
```

Comment:
Make this match the single-date input version

Reply:
4316e7f